priv/repo/migrations/20230314073042_add_unique_index_to_new_contact_id_for_users.exs

```diff
@@ -0,0 +1,7 @@
+defmodule Glific.Repo.Migrations.AddUniqueIndexToNewContactIdForUsers do
+  use Ecto.Migration
+
+  def change do
+    create unique_index(:users, [:contact_id], name: "unique_contact_id_index")
```

Comment:
can we use create if not exists ?

Reply:
@pankaj-ag added as per the suggestion, please check